Cargo.toml

```diff
@@ -17,13 +17,9 @@ azure-devops = { project = "jonhoo/jonhoo", pipeline = "openssh", build = "23" }
 codecov = { repository = "jonhoo/openssh-rs", branch = "master", service = "github" }
 maintenance = { status = "experimental" }
 
-[dependencies]
-tempfile = "3.1.0"
-shell-escape = "0.1"
-tokio = { version = "1", features = [ "process", "io-util" ] }
-
-[dev-dependencies]
-lazy_static = "1.4.0"
-regex = "1"
-tokio = { version = "1", features = [ "full" ] }
+[workspace]
+members = ["process_impl", "mux_client_impl"]
 
+[dependencies]
+process_impl = { version = "0.1.0", path = "process_impl" }
+mux_client_impl = { version = "0.1.0", path = "mux_client_impl", optional = true }
```

Comment:
I'd prefer to not split these into crates, because it would require that we also publish each of those to crates.io as well. Let's instead make them modules of the main crate.

Reply:
These two workspaces are now merged back into `src/`.